pages/common/vault.md

```diff
@@ -23,6 +23,10 @@
 
 `vault read secret/{{hello}}`
 
+- Read a specific field from the value:
+
+`vault read -field={{field_name}} secret/{{hello}}`
```

Comment:
Is there a dash missing here?
```suggestion
`vault read --field={{field_name}} secret/{{hello}}`
```

Reply:
Not at all. Official docs: https://www.vaultproject.io/docs/commands/read/#output-options.